### PR TITLE
Remove unused GA3 UA code

### PR DESCRIPTION
**Pull request by briowill**
Related to #https://github.com/Automattic/martech/issues/2318

## Proposed Changes

Google Universal Analytics (GA3 UA) has been deprecated and we have updated to Google Analytics 4. 
This PR Removes unused GA3 UA code.

## Testing Instructions

1. Checkout branch 
TBD

### CODE REVIEW

File: client/lib/analytics/ad-tracking/record-order.ts [hunk 1]
```diff
@@ -590,29 +589,6 @@ function recordOrderInJetpackGA(
...
-						id: product_id.toString(),
-						name: product_name_en.toString(),
-						quantity: parseInt( String( volume ) ),
-						price: ( costToUSD( cost, cart.currency ) ?? '' ).toString(),
-						brand: GA_PRODUCT_BRAND_JETPACK,
-					} )
-				),
-			},
-		];
-		debug( 'recordOrderInJetpackGA: Record Jetpack Purchase', jetpackParams );
```
Comment (gmovr) [inline client/lib/analytics/ad-tracking/record-order.ts:614]: This involves GA4, which we need to keep. We only want to get rid of the calls to GA3/UA.

File: client/lib/analytics/ad-tracking/record-order.ts [hunk 2]
```diff
@@ -633,29 +609,6 @@ function recordOrderInAkismetGA(
...
-						name: product_name_en.toString(),
-						quantity: parseInt( String( volume ) ),
-						price: ( costToUSD( cost, cart.currency ) ?? '' ).toString(),
-						brand: GA_PRODUCT_BRAND_AKISMET,
-					} )
-				),
-			},
-		];
-		debug( 'recordOrderInAkismetGA: Record Akismet Purchase', akismetParams );
-		window.gtag( ...akismetParams );
```
Comment (gmovr) [inline client/lib/analytics/ad-tracking/record-order.ts:658]: This is also GA4.

File: client/lib/analytics/ad-tracking/record-order.ts (hunk 1 above)
Comment (briowill) [reply client/lib/analytics/ad-tracking/record-order.ts:614]: @gmovr jetpackparms is using jetpackGoogleAnalyticsGtag: 'UA-52447-43' as the send to id. Isn't 'UA-52447-43' an old UA ? It is a similar case with akismetParams below.

File: apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 window.configData = {
 	env_id: 'production',
 	i18n_default_locale_slug: 'en',
-	google_analytics_key: 'UA-10673494-15',
```
Comment (gmovr) [inline apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js:4]: I don't see this required anywhere, but I will check again. To test ETK, we'd need to build and sync it. We could also ask someone more familiar with ETK.

File: client/lib/analytics/ad-tracking/record-order.ts (hunk 1 above)
Comment (gmovr) [reply client/lib/analytics/ad-tracking/record-order.ts:614]: You're right! My bad. D:  

GA4 is handled separately in `recordOrderInJetpackGA`, `recordOrderInWPcomGA4` etc. cc: @robertsreberski for a quick look at this removal code as it affects Akismet and Jetpack.

File: client/lib/analytics/ad-tracking/load-tracking-scripts.js [hunk 4]
```diff
@@ -65,7 +64,7 @@ function getTrackingScriptsToLoad() {
 
 	// The Gtag script needs to be loaded with an ID in the URL so we search for the first available one.
 	const enabledGtags = [
-		mayWeTrackByTracker( 'ga' ) && getGaGtag(),
+		mayWeTrackByTracker( 'ga' ),
```
Comment (gmovr) [inline client/lib/analytics/ad-tracking/load-tracking-scripts.js]: We can remove this whole line 👍 
```suggestion
```

**Review by gmovr — COMMENTED** [review]
I did another test now as I had a bit more time. Left some comments. 

It seems to be working well, I don't see calls to UA anymore, and the page views are sending correctly to the WP GA4 account. I also tested an order, signup, etc., and those events are working.

@robertsreberski  -- Could you take a look? Since we're also removing the calls for Jetpack and Aksimet UA. Also, if you know any other things we'd need to check?

File: client/lib/analytics/ad-tracking/record-order.ts (hunk 1 above)
Comment (robertsreberski) [reply client/lib/analytics/ad-tracking/record-order.ts:614]: Confirming that it's redundant

**Review by robertsreberski — COMMENTED** [review]
These are some notes I've taken when looking through the code:
- We should also get rid of GA Enhanced Ecommerce
- We can remove unit tests under `test/google-analytics`
- We could make the cleanup "cleaner" by removing `ad-tracking/google-analytics` and refactoring code to refer to `ad-tracking/google-analytics-4`

The PR looks good from the point of Jetpack/Akismet!

**Comment by briowill** [comment]
> These are some notes I've taken when looking through the code:
> 
> * We should also get rid of GA Enhanced Ecommerce
> * We can remove unit tests under `test/google-analytics`
> * We could make the cleanup "cleaner" by removing `ad-tracking/google-analytics` and refactoring code to refer to `ad-tracking/google-analytics-4`
> 
I looked through the code and will be handling the "cleaner" cleanup in a different PR. It involves more changes that I initially anticipated.

**Comment by robertsreberski** [comment]
I think it might be good to remove Unit tests in this PR. Other than this code looks ok and doesn't impact Jetpack/Akismet.

**Comment by briowill** [comment]
> These are some notes I've taken when looking through the code:
> 
> * We should also get rid of GA Enhanced Ecommerce
> * We can remove unit tests under `test/google-analytics`
> * We could make the cleanup "cleaner" by removing `ad-tracking/google-analytics` and refactoring code to refer to `ad-tracking/google-analytics-4`
> 
> The PR looks good from the point of Jetpack/Akismet!

New Issue tracking changes https://github.com/orgs/Automattic/projects/388/views/1?pane=issue&itemId=66622673

**Comment by briowill** [comment]
Follow up PR https://github.com/Automattic/wp-calypso/pull/91857

**Pull request merged**